.gitlab-ci.yml

```diff
@@ -243,6 +246,20 @@ publish:s3:apt-repo:
     -   cp ${CI_PROJECT_DIR}/output/mender-configure*.deb pool/main/m/mender-configure/
     - fi
 
+    # mender-monitor
+    - if is_newer_version mender-monitor ${MENDER_MONITOR_VERSION}; then
+    -   if is_final_tag ${MENDER_MONITOR_VERSION}; then
+    -     codename=stable
+    -   else
+    -     codename=experimental
+    -   fi
+    -   for change_file in $(ls ${CI_PROJECT_DIR}/output/mender-monitor_*.changes); do
+    -     reprepro --keepunreferencedfiles include ${codename} ${change_file}
+    -   done
+    - else
+    -   cp ${CI_PROJECT_DIR}/output/mender-monitor*.deb pool/main/m/mender-monitor/
```

Comment:
Not too familiar with the s3 side of this, is this really the right place to put it :thinking: ?

Reply:
Discussed in person - No, this package needs to go into a different bucket `mender-binaries`.